components/input/Input.tsx

```diff
@@ -92,6 +94,13 @@ export default class Input extends React.Component<InputProps, any> {
     }
   }
 
+  handleKeyUp = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    const { onKeyUp } = this.props;
+    if (onKeyUp) {
+      onKeyUp(e);
+    }
+  }
+
```

Comment:
I don't think `handleKeyUp` is neccessary, `otherProps` will handle it.

Reply:
You're right thanks @afc163